src/structures/ApplicationCommand.js

```diff
@@ -77,6 +77,12 @@ class ApplicationCommand extends Base {
      * @type {boolean}
      */
     this.defaultPermission = data.default_permission;
+
+    /**
+     * Autoincrementing version identifier updated during substantial record changes
```

Comment:
Isn’t this unnecessarily long?
```suggestion
     * Version identifier updated every time the command is edited
```

Reply:
That was added by night after some disagreements around the few initial descriptions, so I think we should keep it.